tests/framework/extensions/provisioning/ssh.go

```diff
@@ -22,14 +22,13 @@ import (
 )
 
 const (
-	cpuUsageVar                                            = 100 // 100 is just a placeholder until we can determine an actual number. Even with cpu usage spiking it should not go past 100% cpu usage and previous issues concerning this were hitting around 130% and above
-	checkCPU                 provisioninginput.SSHTestCase = "CheckCPU"
-	checkCPUCommand                                        = "ps -C agent -o %cpu --no-header"
-	nodeReboot               provisioninginput.SSHTestCase = "NodeReboot"
-	activeState                                            = "active"
-	runningState                                           = "running"
-	machineSteveResourceType                               = "cluster.x-k8s.io.machine"
```

Comment:
ssh.go does need the machineSteveResourceType?

Reply:
Yes it does

https://github.com/rancher/rancher/blob/f338c671ba4b81e8ab437b20d5baff478fb74df6/tests/framework/extensions/provisioning/ssh.go#L63